.github/workflows/presubmit.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [4.1, 4.2, AF.3.1, 2020.4, 2021.1]
+        version: [AF.3.1, 2020.4, 2021.1]
```

Comment:
Android Studio 4.2 is the current stable release, so we need to support it.

Reply:
Ah, this was inadvertent; I'll re-add.